CMakeLists.txt

```diff
@@ -39,8 +39,100 @@ cpr_option(CMAKE_USE_OPENSSL "Use OpenSSL code. Experimental" ON)
 message(STATUS "=======================================================")
 
 include(GNUInstallDirs)
+include(FetchContent)
+include(cmake/code_coverage.cmake)
+
+# Curl configuration
+if(USE_SYSTEM_CURL)
+    find_package(CURL)
```

Comment:
Thanks for the PR.
Could you please add some output, once `FindCURL` has finished depending on what the result of `CURL_FOUND` is.

Something like:
`True` -> System CURL found
`ELSE` -> System CURL not found and then fail

Reply:
FindPackage prints such things by default, if curl is not found cpr will try to use fetch content